rule-engine/rule-engine-components/src/test/java/org/thingsboard/rule/engine/telemetry/TbMsgAttributesNodeTest.java

```diff
@@ -180,6 +204,97 @@ void givenNotifyDeviceMdValue_whenSaveAndNotify_thenVerifyExpectedArgumentForNot
         assertThat(notifyDevice).isEqualTo(expectedArgumentValue);
     }
 
+    @Test
+    void givenUseTemplateTrueAndNoMetadataKeyInMsg_whenOnMsg_thenSaveAttributes() throws Exception {
+        config.setUseAttributesScopeTemplate(true);
+        config.setScope("${scopeInMetadata}");
+        config.setUpdateAttributesOnlyOnValueChange(false);
+        nodeConfiguration = new TbNodeConfiguration(JacksonUtil.valueToTree(config));
+        node.init(ctx, nodeConfiguration);
+
+        final Map<String, String> mdMap = Map.of("scope", "SERVER_SCOPE");
+        final String data = "{\"TestAttribute_2\": \"humidity\", \"TestAttribute_3\": \"voltage\"}";
+        TbMsg msg = TbMsg.newMsg(TbMsgType.POST_ATTRIBUTES_REQUEST, deviceId, new TbMsgMetaData(mdMap), data);
+
+        node.onMsg(ctx, msg);
+
+        verify(telemetryService).saveAndNotify(eq(ctx.getTenantId()), eq(msg.getOriginator()), eq(AttributeScope.SERVER_SCOPE),
+                anyList(), eq(true), eq(new TelemetryNodeCallback(ctx, msg)));
+        verify(ctx).tellSuccess(eq(msg));
+    }
+
+    @Test
+    void givenUseTemplateTrueAndStrangeDataKey_whenOnMsg_thenSaveAttributes() throws Exception {
```

Comment:
```suggestion
    void givenUseTemplateTrueAndSpongeCaseDataKey_whenOnMsg_thenSaveAttributes() throws Exception {
```
![image](https://github.com/thingsboard/thingsboard/assets/16356590/c5ee51a0-846a-488f-b52f-951e5b15cb3c)

Reply:
you might call it "SpongeCase," drawing from the internet meme "Mocking SpongeBob," where text is typed with alternating uppercase and lowercase letters to convey a mocking tone.